const CustomError = require("./../utils/custom-error");

class TodoService {

  async create(data) {
    // create
    return null
  }

  async getAll() {
    // get all
    return null
  }

  async getOne(todoId) {
    // get one
    return null
  }

  async update(todoId, data) {
    // update
    return null
  }

  async delete(todoId) {
    // delete
    return null
  }

}

module.exports = new TodoService();